app/scripts/metamask-controller.js

```diff
@@ -4876,7 +4876,7 @@ export default class MetamaskController extends EventEmitter {
 
   async attemptLedgerTransportCreation() {
     return await this.#withKeyringForDevice(
-      HardwareDeviceNames.ledger,
+      { name: HardwareDeviceNames.ledger },
```

Comment:
For the next reviewer: Looks like this was missing on this function. All other functions that uses `this.#withKeyringForDevice` were using "named parameters" already.